JS/js-forms/js-forms_checkbox-input/js/index.js

```diff
@@ -2,6 +2,7 @@ console.clear();
 
 const form = document.querySelector('[data-js="form"]');
 const tosError = document.querySelector('[data-js="tos-error"]');
+const success = document.querySelector('[data-js="success"]');
```

Comment:
Could be a tad more explicit imo:
```suggestion
const successMessage = document.querySelector('[data-js="success-message"]');
```